testCourse/courseInstances/Sp15/assessments/exam9-sequentialQuestions/infoAssessment.json

```diff
@@ -0,0 +1,47 @@
+{
+    "uuid": "d7bcc376-4f23-41d6-9f71-87dd1d23991b",
+    "allowAccess": [{"mode": "Public", "credit": 100}],
+
+    "title": "Force Student to Complete Questions In-Order",
+    "type": "Exam",
+    "set": "Exam",
+    "number": "9",
+
+    "minAdvancePerc": 100,
+    "zones": [
```

Comment:
We only have question-level testing for now. It would be great if we can add minAdvancePerc tests at assessment, zone, and alternative group levels

Reply:
Thanks for pointing that out, I forgot to verify that each higher config level was covered when I re-arranged the testing assessment at the last minute. I'll add this to my final to-do.